SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2867,7 +2887,11 @@ void USpatialNetDriver::RemoveActorChannel(Worker_EntityId EntityId, USpatialAct
 		return;
 	}
 
-	EntityToActorChannel.FindAndRemoveChecked(EntityId);
+	const int32 ValuesRemoved = EntityToActorChannel.Remove(EntityId);
```

Comment:
this change isn't necessary - we checked about for validity

Reply:
I agree the ensure isn't necessary. Changing to just be a Remove rather than FindAndRemoveChecked.